docs/Using-Fleet/Permissions.md

```diff
@@ -12,45 +12,43 @@ Users with the Admin role receive all permissions.
-| Run saved queries as live queries against all hosts  | ✅       | ✅         | ✅    |
-| Run custom queries as live queries against all hosts |          | ✅         | ✅    |
 | Enroll hosts                                         |          | ✅         | ✅    |
 | Delete hosts                                         |          | ✅         | ✅    |
 | Transfer hosts between teams\*                       |          | ✅         | ✅    |
-| Create saved queries                                 |          | ✅         | ✅    |
-| Edit saved queries                                   |          | ✅         | ✅    |
-| Delete saved queries                                 |          | ✅         | ✅    |
-| Schedule queries for all hosts                       |          | ✅         | ✅    |
-| Schedule queries for all hosts assigned to a team\*  |          | ✅         | ✅    |
-| Create packs                                         |          | ✅         | ✅    |
-| Edit packs                                           |          | ✅         | ✅    |
-| Delete packs                                         |          | ✅         | ✅    |
-| Create labels                                        |          | ✅         | ✅    |
-| Edit labels                                          |          | ✅         | ✅    |
-| Delete labels                                        |          | ✅         | ✅    |
-| Add policies for all hosts                           |          | ✅         | ✅    |
-| Remove policies for all hosts                        |          | ✅         | ✅    |
-| Add policies for hosts assigned to team\*            |          | ✅         | ✅    |
-| Remove policies for hosts assigned to team\*         |          | ✅         | ✅    |
-| Create users                                         |          |            | ✅    |
-| Edit users                                           |          |            | ✅    |
-| Delete users                                         |          |            | ✅    |
-| Edit own user information                            | ✅       | ✅         | ✅    |
+| Create, edit, and delete labels                      |          | ✅         | ✅    |
+| Browse all software                                  | ✅       | ✅         | ✅    |
+| Filter software by vulnerabilities                   | ✅       | ✅         | ✅    |
+| Filter hosts by software                             | ✅       | ✅         | ✅    |
+| Filter software by team*                             | ✅       | ✅         | ✅    |
+| Manage vulnerability automations         |          | ✅         | ✅    |
+| Run saved queries as live queries against all hosts  | ✅       | ✅         | ✅    |
+| Run custom queries as live queries against all hosts |          | ✅         | ✅    |
+| Create, edit, and delete queries                     |          | ✅         | ✅    |
+| Create, edit, and delete scheduled queries for all hosts |          | ✅         | ✅    |
```

Comment:
```suggestion
| Create, edit, view, and delete scheduled queries for all hosts |          | ✅         | ✅    |
```

Reply:
TODO: Add a new "View all queries" line with checks for observer, maintainer, and admins.